minds/utils.py

```diff
@@ -0,0 +1,15 @@
+from urllib.parse import urlparse, urlunparse
+
+def create_base_url_openai(base_url: str) -> str:
```

Comment:
maybe rename to `get_openai_base_url` or `gen_openai_base_url`

Reply:
Done